README.md

```diff
@@ -40,23 +46,27 @@ $ cd my-git-project
 
 This command moves into the newly created directory.
 
-Now that we're in the directory where we want Git to watch for changes (adding,
-removing, and editing files), let's set up this directory by _initializing_ it.
-In the terminal type `git init`. It should look like this:
+Now that we're **in the directory** where we want Git to watch for changes
+(adding, removing, and editing files), we next need to _initialize_ this
+directory as a Git repo. We only need to do this step once for each project we
+want to track.
+
+In the terminal type `git init`. It should look something like this:
 
 ```console
 $ git init
 Initialized empty Git repository in /Users/avi/my-git-project/.git/
 ```
 
-Git lets us know that it has put `/Users/avi/my-git-project` under its protection.
-Git also tells us that it stores its own data in the `.git` directory. This hidden
-directory, `.git`, is where Git keeps important stuff, like the commit
-history. Don't go in there and start randomly deleting things! That said, if ever
-you do `git init` in the wrong directory, you can `rm -rf .git` and return the
-directory to a plain-old, unprotected directory.
+The message above lets us know that our new directory is now being tracked by
+Git. It also shows that a new subfolder `.git` has been created. This hidden
+directory is where Git keeps important stuff, like the commit history. Don't go
+in there and start randomly deleting things! That said, if ever you do `git
+init` in the wrong directory, you can `rm -rf .git` to delete the `.git` folder
+and all its contents and return the directory to a plain-old, unprotected
+directory.
 
-Be careful about making an entire directory, like our home directory or our
+Be careful about making a containing directory, like our home directory or our
 desktop, into a Git repository accidentally. Make sure you only type `git init`
 _within_ the directory you want `git` to track.
```

Comment:
This may be worth emphasizing strongly, I've seen many cases where students DO initialize an empty git repo in their home directory and it creates so, so many problems...